stylesheets/index.css

```diff
@@ -0,0 +1,82 @@
+.container {
+    width: 1400px;
+    height: 800px;
+    display: grid;
+    grid-template-columns: repeat(4, 1fr);
+    grid-template-rows: repeat(4, 1fr) ; 
+}
+
+#headbar {
+   
+    grid-column: 1/5;
+    grid-row: 1/2;
+
+    }
+    
+#navbar{
+   
+    grid-column: 1/span 1;
+    grid-row: 2/5;
+}
+
+#mainbar{
+   
+    grid-column: 2/5;
+    grid-row: 2/4;
+ 
+}
+
+#footerbar{
+  
+    grid-column: 2/span 3;
+    grid-row: 4/5;
+    align-self: end ;
+    
+}
+
+a:hover {
+    background-color: darkgreen;
+   
+ }
+
+footer {
+    padding-top: 120px;
+}
```

Comment:
Spacing inside of rule sets across the CSS files is inconsistent, I suggest removing any blank lines inside of rule sets then keeping a single blank line between each rule set.